### DISCUSSION TITLE
磊科（netcore）N60 AX6000有没有支持计划。CPU：mt7986av

### DISCUSSION THREAD

**Discussion by jwkinko** [immortalwrt/immortalwrt#1172] [General]
[]()https://www.netcoretec.com/product-center/family/wifi-router/2659.html

![n60_01.jpg](https://github.com/immortalwrt/immortalwrt/assets/95012013/2d6b8649-c142-42c2-9f05-1690db3b3c86)

**Comment by neobar** [comment]
https://github.com/openwrt/openwrt/commit/4ae474afbd023a3386f4276ca8f948a44b49ff1b
